python/dgl/heterograph_index.py

```diff
@@ -149,6 +150,7 @@ def ctx(self):
         """
         return _CAPI_DGLHeteroContext(self)
 
+    @utils.cached_member(cache='_cache', prefix='nbits')
```

Comment:
Why cache it? Actually, shouldn't we just remove it?

Reply:
Do you mean only `gidx.dtype` should be preserved?

Reply:
Yes. I suppose `gidx.dtype` covers `gidx.nbits`.